apis/appmesh/v1beta2/gatewayroute_types.go

```diff
@@ -60,14 +89,105 @@ type GRPCGatewayRoute struct {
 
 // HTTPGatewayRouteMatch refers to https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_gateways.html
 type HTTPGatewayRouteMatch struct {
+	// Either Prefix or Hostname must be specified. Both are allowed as well.
 	// Specifies the path to match requests with
-	Prefix *string `json:"prefix"`
+	// +optional
```

Comment:
Is this correct? Every option under this struct is optional??

Reply:
Yes, we need either prefix or hostname so that validation is done by admission webhook